SAKPaaS/src/assets/i18n/en.json

```diff
@@ -35,61 +35,61 @@
     },
     "high": {
       "checkout-time": "Long waiting time",
-      "safety-distance": "Safety distance of 1.5m can't be guaranteed.",
+      "safety-distance": "Safe distance of 1.5m can't be guaranteed.",
```

Comment:
Ist safe distance überhaupt englisch? ^^ Ich glaub das muss echt safety distance heißen

Reply:
```Safe distance``` is the correct term.

Reply:
![image](https://user-images.githubusercontent.com/43894937/80273047-4b6cdf80-86cf-11ea-9f46-db3bff978ef0.png)

Reply:
Internet sagt, beides ist richtig ^^